src/routes/(app)/namespaces/[namespace]/workflows/[workflow]/[run]/stack-trace/+page.svelte

```diff
@@ -1,16 +1,20 @@
 <script lang="ts">
   import { page } from '$app/stores';
+  import { settings } from '$lib/stores/settings';
 
   import WorkflowStackTrace from '$lib/pages/workflow-stack-trace.svelte';
 
   import PageTitle from '$lib/components/page-title.svelte';
   import WorkflowRunLayout from '$lib/layouts/workflow-run-layout.svelte';
 
   const workflow = $page.params.workflow;
-  $: isCloud = $page.data?.settings?.runtimeEnvironment?.isCloud;
 </script>
 
 <PageTitle title={`Stack Trace | ${workflow}`} url={$page.url.href} />
-<WorkflowRunLayout cancelEnabled={!isCloud} signalEnabled={!isCloud}>
+<WorkflowRunLayout
+  cancelEnabled={!$settings.workflowCancelDisabled}
+  signalEnabled={!$settings.workflowSignalDisabled}
```

Comment:
Do we still need the isCloud check here? Like how would we turn on workflowSignalDisabled for cloud folks?

Reply:
Yep good call, fixed in [13cbc10](https://github.com/temporalio/ui/pull/1037/commits/13cbc108164982da1141265301ba78b9bbb6eee4)

Reply:
Nice this looks real good now.